backend/src/kube.rs

```diff
@@ -0,0 +1,5 @@
+
+struct Kube {
```

Comment:
This should be `pub struct` if you want to make it be used in other files (which we want).